src/components/Layout/Layout.tsx

```diff
@@ -171,32 +175,17 @@ export const Layout = ({
       menuButtonRef?.current?.focus();
     }
   };
-  let platformNav = <div className="layout-sidebar-platform">
-    <PlatformNavigator
-      currentPlatform={
-        currentPlatform
-      }
-      isPrev={isPrev}
-    />
-  </div>
-  let menu = <Menu
-    currentPlatform={currentPlatform}
-    rootMenuNode={rootMenuNode}
-  />
+
+  let menu = (
+    <Menu currentPlatform={currentPlatform} rootMenuNode={rootMenuNode} />
+  );
   if (isGen2) {
-    menu = <Menu
-      rootMenuNode={rootMenuNode}
-    />
-    platformNav = null;
```

Comment:
This line `platformNav = null` was throwing a type error for me. I think it's ok that we just conditionally render it in the Layout below though? Via:
```
{isGen2 ? null : (
    <div className="layout-sidebar-platform">
      <PlatformNavigator
        currentPlatform={currentPlatform}
        isPrev={isPrev}
      />
    </div>
  )}
```